clang/tools/scan-build/bin/scan-build

```diff
@@ -1465,6 +1466,17 @@ sub HtmlEscape {
   return $tmp;
 }
 
+##----------------------------------------------------------------------------##
+# URLEscape - encode characters that are special in URLs
+##----------------------------------------------------------------------------##
+
+sub URLEscape {
+  my $arg = shift || '';
+  my $tmp = $arg;
+  $tmp =~ s/\+/%2B/g;
```

Comment:
Why is an additional variable `$tmp` needed? Can't this just be `$arg =~ s/\+/%2B/g;`?

Reply:
Good point! Will update this before landing.